package-system/libpng/build_config.json

```diff
@@ -109,6 +109,19 @@
                 "custom_test_cmd": [
                     "./test_png_linux.sh"
                 ]
+            },
+            "Linux-aarch64": {
+                "depends_on_packages" :[
+                    ["zlib-1.2.11-rev5-linux-aarch64", "ce9d1ed2883d77ffc69c7982c078595c1f89ca55ec19d89fe7e6beb05f774775", ""]
+                ],
+                "cmake_generate_args_release": [
+                    "-G",
+                    "Unix\\ Makefiles",
```

Comment:
We used `Ninja` for the other 3rdParty libraries to build.
For example SPIRV is using [Ninja](https://github.com/o3de/3p-package-source/blob/4cf88caf60726d140d9707801e5cdb4b7ba0aa9c/package-system/SPIRVCross/build_config.json#L81).

We should stop mixing CMake generators across our Linux 3rdParty builds